priv/perf/apps/omg_load_testing/lib/connection/utils.ex

```diff
@@ -1,4 +1,4 @@
-defmodule LoadTest.Connection.Utils do
+defmodule OMG.LoadTesting.Connection.Utils do
```

Comment:
any particular reason we're going for the *omg* prefix?

Reply:
Just wanted to stick to the convention from the umbrella app, but now I see the convention from cabbage tests.